### DISCUSSION TITLE
Trigger plot change

### DISCUSSION THREAD

**Discussion by ran-codes** [qmd-lab/closeread#87] [Q&A] [answered]
Any examples of triggers modifying a single plot rather than showing a different plot?

I have made scrollies like this in JS. So would this be possible in R cr-chunks or ojs cr-chunks? (if so can you share some examples?)

Thank you for the great package!!

**Comment by ran-codes** [comment, marked as answer]
nvm i found this https://closeread.netlify.app/gallery/demos/ojs-variables/.

(but still curious is this even possible in R visualization libraries)?

**Reply by jimjam-slam** [reply]
Hey @ran-codes! My feeling is that it would be difficult to do this with most R-based visualisation libraries, unless they had some method of JavaScript-based communication built-in.

That said, if you're working in R, Python or Julia, you can do some pretty convincing effects user by swapping out plot images! I'll have a demo up of this in a few weeks with ggplot2 😊

**Reply by ran-codes** [reply]
Thanks James !! (Also - really excited to see how this can interact with Sverto; I used to make pure svelte scrollies - and it would be awesome if there was a streamline that logic into closeread quarto doc)

**Reply by jimjam-slam** [reply]
I've definitely worked on the OJS side of Closeread with integrations like Sverto in mind! I'd say leave the scrolling logic side of things to Closeread and focus on how the graphics will change: you ought to be able to drive Svelte props with Closeread variables directly using accessor syntax as usual 😊
